libs/packages/components/package.json

```diff
@@ -11,8 +11,8 @@
   },
   "homepage": "https://github.com/GSA/sam-design-system/tree/master/libs/packages/components",
   "peerDependencies": {
-    "@angular/common": ">=14.2.12 <15.0.0",
-    "@angular/core": ">=14.2.12 <15.0.0",
+    "@angular/common": ">=14.2.12 <15.3.0",
```

Comment:
Maybe update this to `>=14.2.12 ^15.0.0` so that if another minor release is made this file does not need to be updated again?

Reply:
updated the peer dependencies